app/src/main/assets/json/server-config.json

```diff
@@ -18073,6 +18376,11 @@
             "close": "Close"
           }
         },
+        "payment_types": {
+          "apple_pay": "Apple Pay",
+          "google_pay": "Google Pay",
```

Comment:
Is Google Play now an available payment method?? Will take a look into the payments side of things as this file is generated by a script 🤔